secure/key-manager/src/tests.rs

```diff
@@ -244,10 +281,10 @@ fn test_consensus_rotation() {
 }
 
 #[test]
-// This verifies the key manager is properly setup
-fn test_key_manager_init() {
+// This verifies the key manager is properly setup and that a
```

Comment:
not finished?

Reply:
well the code is still incomplete too :P